documentation/asciidoc/topics/ref_batch_operations.adoc

```diff
@@ -0,0 +1,105 @@
+[id='batch-operations_{context}']
+= Example Batch Operations
```

Comment:
@rigazilla and @ryanemerson Hey guys, I've added some example batch operations to this docs PR. I've covered users, caches, counters, schema, and tasks. These are the things you see in the console so it made sense to me to show some batch scripts for those resources. 

Are there other examples related to cluster initialization or possibly upgrade that we could include here?

Reply:
This is really good :+1: 

The only other example I can think of is creating a cache template from a xml file and then creating caches in the batch using that template.